core/string/ustring.cpp

```diff
@@ -64,6 +64,15 @@ const char16_t Char16String::_null = 0;
 const char32_t String::_null = 0;
 const char32_t String::_replacement_char = 0xfffd;
 
+// strlen equivalent function for char32_t * arguments.
+size_t strlen(const char32_t *p_str) {
```

Comment:
```suggestion
_FORCE_INLINE_ size_t strlen(const char32_t *p_str) {
```

Reply:
Ok, I guess we can force-inline it. I don't have a strong opinion about this.